File: Java/LinkedList/LinkedList.java
/**
 * Main class of the Java program.
 */

public class Main {
    
    
    // Linked list implementation in Java

    public static class LinkedList {
      // Creating a node
      Node head;
    }
     public static class Node {
        int value;
        Node next;
    
        Node(int d) {
          value = d;
          next = null;
        }
    }

    public static void main(String[] args) {
        LinkedList linkedList = new LinkedList();

        // Assign value values
        linkedList.head = new Node(1);
        Node second = new Node(2);
        Node third = new Node(3);
    
        // Connect nodess
        linkedList.head.next = second;
        second.next = third;
    
        // printing node-value
        while (linkedList.head != null) {
          System.out.print(linkedList.head.value + " ");
          linkedList.head = linkedList.head.next;
        }
    }
}
/* 
Output: 
1 2 3  
*/

File: C++/LinkedList/LinkedList.cpp
// Main function of the C++ program.

#include <bits/stdc++.h>
using namespace std;

// Creating a node
class Node {
  public:
  int value;
  Node* next;
  Node(int v) {
      value = v;
      next = nullptr;
  }
};


int main()
{

    Node* one = new Node(1);
    Node* two = new Node(2);
    Node* three = new Node(3);
    one->next = two;
    two->next = three;

    // print the linked list value
     Node* head = one;
     while (head != nullptr) {
        cout << head->value << " ";
        head = head->next;
      }

    return 0;
}
/* 
Output: 
1 2 3 
*/
